.github/actions/setup-sentry/action.yml

```diff
@@ -127,14 +126,22 @@ runs:
         echo "acceptance-dir=.artifacts/visual-snapshots/acceptance" >> "$GITHUB_OUTPUT"
 
     - name: Install python dependencies
+      if: steps.venv.outputs.cache-hit != 'true'
+      shell: bash
+      env:
+        WORKDIR: ${{ inputs.workdir }}
+      run: |
+        cd "$WORKDIR"
+        pip install -r requirements-dev-frozen.txt
```

Comment:
I would put this next to the venv step above -- you can also use `workdir` directly I believe rather than needing to `cd`